.github/workflows/maestro_operator-test.yaml

```diff
@@ -74,14 +82,35 @@ jobs:
           echo "Pod is created"
         fi
 
-    - name: Send request
+    - name: Wait for pod to be ready
       run: |
-        curl -L "http://127.0.0.1:30051/?Prompt=Los+angels"
+        kubectl wait --for=condition=ready pod -l app.kubernetes.io/instance=weather-checker-ai --timeout=60s
 
+    - name: Port forward and send request
+      run: |
+        # Start port-forward in background
+        kubectl port-forward svc/weather-checker-ai 30051:80 &
```

Comment:
In past experience, I've never had any luck trying to port-forward in a github action. Let me see what we used in serverless instead...

Reply:
we hacked the svc to make it a load balancer and then directly hit that

```
kubectl patch svc gateway -p '{"spec": {"type": "LoadBalancer"}}'
export GATEWAY_HOST="http://$(kubectl get svc gateway -o jsonpath="{.status.loadBalancer.ingress[0].ip}"):8000"
```

we then used `$GATEWAY_HOST` when we had to send API requests

Reply:
interesting. Let me try one more thing since Aki had it working before.

Reply:
OK trying your suggestion.